File: JanperCab/5-JanperCab/ClientApp/src/app/_models/enquiry/EnquiryForInvoicingDto.ts
export class EnquiryForInvoicingDto {
  id: number;
  jobType: string;
  deliveredTime: Date;
  customerName: string;
  gstRate: number;
  discountRate: number;
  subTotal: number;
  totalGst: number;
  totalPrice: number;
}

File: JanperCab/3-Application/Dtos/Enquiry/EnquiryForInvoicingDto.cs
using System;

namespace _3_Application.Dtos.Enquiry
{
    public class EnquiryForInvoicingDto
    {
        public int Id { get; set; }

        public string JobType { get; set; }

        public DateTime? DeliveredTime { get; set; }

        public string CustomerName { get; set; }

        public decimal GstRate { get; set; }

        public decimal DiscountRate { get; set; }

        public decimal SubTotal { get; set; }

        public decimal TotalGst { get; set; }

        public decimal TotalPrice { get; set; }
    }
}